docs/faq.md

```diff
@@ -0,0 +1,89 @@
+# Frequently Asked Questions
+
+## What is Jupytext?
+
+Jupytext is a Python package that can convert Jupyter notebooks to scripts or Markdown documents. It can also convert these text documents back to Jupyter notebooks.
+
+## Why would I want to convert my notebooks to text?
+
+The text representation have much cleaner diffs than the original notebook format. Merging multiple contributions to a notebook in any of these text formats is easier than with the JSON format. Last but not least, acting on a notebook represented as text (spell check, reformat, ...) is sometimes more comfortable than in Jupyter.
+
+## How do I use Jupytext?
+
+Open the notebook you want to version control. _Pair_ the notebook to a script or a Markdown file using either the [Jupytext Menu](https://github.com/mwouts/jupytext/blob/master/README.md#jupytext-menu-in-jupyter-notebook) in Jupyter Notebook or the [Jupytext Commands](https://github.com/mwouts/jupytext/blob/master/README.md#jupytext-commands-in-jupyterlab) in JupyterLab.
+
+Save the notebook, and you get two copies of the notebook: the original `*.ipynb` file, together with its paired text representation.
+
+## Which Jupytext format do you recommend?
+
+I tend to use the Markdown format for notebooks that contain more text than code, as Markdown documents are conveniently edited in IDEs and also well rendered on GitHub.
+
+Saving notebooks as scripts is a convenient choice when you want to refactor your notebook in an IDE (or import it in another notebook, etc). Use the `percent` format if you prefer to get explicit cell markers (compatible with VScode, PyCharm, Spyder, Hydrogen...). If you prefer to get the minimal amount of cell markers, go for the `light` format.
+
+## Can I see a sample of each format?
+
+Go to [our demo folder](https://github.com/mwouts/jupytext/tree/master/demo) and see how our sample `World population` notebook is represented in each format.
+
+## Can I edit the paired text file?
+
+Yes! And when you're done, refresh the notebook in Jupyter. Refreshing will bring the latest changes to your notebook.
+
+## How do paired notebooks work?
+
+The `.ipynb` file contains the full notebook. The paired text file only contains the input cells and selected metadata. When the notebook is loaded by Jupyter, input cells are loaded from the text file, while the output cells and the filtered metadata are restored using the `.ipynb` file.
+
+## Can I create a notebook from a text file?
```

Comment:
Again for content like these, I'd rather make sure there is an explicit section in the "using jupytext" documentation and then link to that from the FAQ...